core/formats/columnstore2.hpp

```diff
@@ -82,45 +82,46 @@ class column final : public irs::column_output {
 
   class address_table {
    public:
+    address_table(ManagedTypedAllocator<uint64_t> alloc) : alloc_{alloc} {
+      offsets_ = alloc_.allocate(kBlockSize);
```

Comment:
Now we allocate very good aligned blocks, so I think it's better utilize allocator.

Reply:
Here would be 2 allocations. One for deque itself and this one for internal contents. Better to benchmark this on test load with many columns.